api/v1/accounts.js

```diff
@@ -22,7 +22,6 @@ const validate = (req, res, next) => {
     }
   });
 };
-
```

Comment:
Please re-add this newline for readability.

Reply:
Omg I didn't mean to resolve this, just about to get started on working on these, thank you for the overlooks.

Reply:
✔